contrib/docs/tutorials/help-im-behind-a-corporate-proxy.md

```diff
@@ -56,3 +58,29 @@ if (process.env.HTTPS_PROXY) {
 ```
 
 4. Start the backend with `yarn start`
+
+## config
+
+If your development environment is in the cloud (like with [AWS Cloud9](https://aws.amazon.com/cloud9/) or an instance of [Theia](https://theia-ide.org/)), you will need to update your configuration.
+
+In `app-config.local.yaml`, you will probably need to make some changes, the exact values will depend on your setup.
```

Comment:
Should just make this `app-config.yaml` imo, or perhaps say there or in a custom config file. The `.local.yaml` one is not committed, so not a great place to put config you care about

Reply:
That's right, personally I fell this sort of configuration is better in an local, git-ignored file but I understand this might be confusing, I've rephrased that.